src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private String color;
+
     public String getRandomColor() {
-        return null;
+        int indexColors = new Random().nextInt(Colors.values().length);
```

Comment:
random should be `private final` class-level field
```suggestion
        int colorIndex = random.nextInt(Colors.values().length);
```